src/pages/Swap/AdvancedSwapMode/Trade/Trade.hooks.ts

```diff
@@ -0,0 +1,46 @@
+import { CSSProperties, useLayoutEffect, useState } from 'react'
+
+enum SizeOfTrade {
+  MIN = 1000,
+  MAX = 100000,
+}
+
+export const useStylingTradeBackground = ({
+  isSell,
+  amountUSDNumber,
+}: {
+  amountUSDNumber: number
+  isSell?: boolean
```

Comment:
isSell is a bit odd, what does that mean?

Reply:
![Selection_202](https://user-images.githubusercontent.com/100695408/192277962-977788d4-fbc3-4425-bb1f-fc973545056c.png)

it is connected to transaction, which token was bought and which was sold